cli/tests/test_sources.rs

```diff
@@ -108,6 +108,7 @@ fn test_create_update_source_custom() {
         "--description=some description",
         "--language=de",
         "--should-translate=true",
+        "--source-type=call",
```

Comment:
If we just modify this test, we don't test creating a source without specifying the type, would it be possible to have a new test that just checks creating a source with `--source-type call` and then source works

Reply:
Yeah, that's fair, but we only have one test with all these conditions. We don't have a test without specifying `language`, `description`, `should-translate` or any of the parameters either.

Not saying that's a good thing, so happy to add one, I'm just also aware that it would be a copy and paste of this test pretty much verbatim. If there is a better way I don't think I'm capable of writing it in reasonable time and would need help.